Gems/EMotionFX/Code/EMotionFX/Rendering/Common/RotateManipulator.cpp

```diff
@@ -367,6 +371,77 @@ namespace MCommon
         }
     }
 
+    bool RotateManipulator::internalCheckIntersectXAxis(
```

Comment:
update: This needs an uppercase name for the member function - `InternalCheckIntersectXAxis`